src/main/java/com/synopsys/integration/alert/channel/azure/boards/web/AzureOauthCallbackController.java

```diff
@@ -62,46 +71,90 @@
     private final AzureBoardsCredentialDataStoreFactory azureBoardsCredentialDataStoreFactory;
     private final ProxyManager proxyManager;
     private final ConfigurationAccessor configurationAccessor;
-    private final ConfigurationFieldModelConverter configFieldModelConverter;
+    private final AzureRedirectUtil azureRedirectUtil;
 
     @Autowired
     public AzureOauthCallbackController(ResponseFactory responseFactory, Gson gson, AzureBoardsChannelKey azureBoardsChannelKey,
```

Comment:
The "a" in "OAuth" is not capitalized in the class name.

Reply:
I have renamed the class and pushed it up with the 'a' capitalized for OAuth.  This PR isn't reflecting that change when I see the files in the browser.